components/classes/experience.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/experience",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Experience",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Digital experience such as a website or social media post.",
+  "definitions": {
+    "experience": {
+      "properties": {
+        "xdm:experience": {
```

Comment:
I will recommend changing this field name to something specific to your use case; "xdm:experience" is very generic